src/cljs/rems/theme.cljs

```diff
@@ -0,0 +1,12 @@
+(ns rems.theme
+  (:require [re-frame.core :as rf]))
+
+(rf/reg-sub :rems.theme/theme (fn [db] (::theme db)))
+(rf/reg-event-db :rems.theme/loaded-theme (fn [db [_ theme]] (assoc db ::theme theme)))
```

Comment:
This could use `::theme`, I think that's OK in its own file.

The reason we had `:theme` before was just that it's kind of a common and standard sub/event for the entire app so it has a "global identity". Now everyone must use the specific namespaced name, which is OK too, just that it is a change.